daemon/src/cfd_actors.rs

```diff
@@ -25,15 +22,11 @@ pub async fn insert_cfd_and_update_feed(
     Ok(())
 }
 
-pub async fn handle_monitoring_event<W>(
+pub async fn handle_monitoring_event(
     event: monitor::Event,
     db: &SqlitePool,
-    wallet: &xtra::Address<W>,
-    projection_address: &xtra::Address<projection::Actor>,
-) -> Result<()>
-where
-    W: xtra::Handler<wallet::TryBroadcastTransaction>,
-{
+    process_manager_address: &xtra::Address<process_manager::Actor>,
```

Comment:
```suggestion
    process_manager: &xtra::Address<process_manager::Actor>,
```

Might make things a bit shorter.